### ISSUE TITLE
Superscript at the end of the string results in error

### ISSUE THREAD

**Issue by illiteratePhD** [ForNeVeR/wpf-math#73]
Trying the following string will yield an error:

```tex
d_{f,t}(u,v) = \sum_{x, y}\left(f(x, y) - t(x - u, y - v)\right)^2
```

**Comment by gsomix** [comment]
Duplicate of #62

**Comment by ForNeVeR** [comment]
Merged into #62.

**Comment by ForNeVeR** [comment]
@illiteratePhD please note that we have fixed the issue with #74.

**Comment by illiteratePhD** [comment]
@ForNeVeR Fantastic! Going to check it out as soon as I am able. Great work guys!
